source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3274,11 +3274,82 @@ export class Media extends CardElement {
 }
 
 export abstract class Input extends CardElement implements IInput {
-    //#region Schema
+    
+    // default label width is 30% of the input container 
+    private static readonly _defaultInputLabelWidth = new SizeAndUnit(30, Enums.SizeUnit.Weight);
 
+    //#region Schema
     static readonly labelProperty = new StringProperty(Versions.v1_3, "label", true);
     static readonly isRequiredProperty = new BoolProperty(Versions.v1_3, "isRequired", false);
     static readonly errorMessageProperty = new StringProperty(Versions.v1_3, "errorMessage", true);
+    static readonly labelWidthProperty = new CustomProperty<SizeAndUnit | undefined>(
+        Versions.v1_6,
+        "labelWidth",
+        (
+            sender: SerializableObject,
+            prop: PropertyDefinition,
+            source: PropertyBag,
+            context: BaseSerializationContext
+        ) => {
+            let result: SizeAndUnit | undefined = prop.defaultValue;
+            const value = source[prop.name];
+            let invalidLabelWidth = false;
+            
+            if (typeof value === "number" && !isNaN(value)) {
+                result = new SizeAndUnit(value, Enums.SizeUnit.Weight);
+                if (result.physicalSize < 0 || result.physicalSize > 100) {
+                    invalidLabelWidth = true;
+                }
+            }
+            else if (typeof value === "string") {
+                try {
+                    result = SizeAndUnit.parse(value);
+                } catch (e) {
+                    invalidLabelWidth = true;
+                }
+            } else {
+                invalidLabelWidth = true;
+            }
+
+            if (invalidLabelWidth) {
+                context.logParseEvent(
+                    sender,
+                    Enums.ValidationEvent.InvalidPropertyValue,
+                    Strings.errors.invalidInputLabelWidth()
+                );
+                result = undefined;
```

Comment:
Why don't we revert to the default value here?

Reply:
So I am setting default here as undefined and when we render the element and label width is undefined, I am using the default value from host config 
![image](https://user-images.githubusercontent.com/12997454/211289342-8ae916d6-1666-447d-8755-bbc6376c3da3.png)

At this place, host config is not available and I want default value to be coming from host config and inside the host config, it has a default value (=30:70) when host does not provide it.